frontend/src/app/globals.css

```
@@ -33,6 +33,19 @@
   }
 }
 
+/* ローディング画面：背景グラデーションをゆっくり斜めに流す */
+@keyframes loadingGradientShift {
```

Stylelint (17.12.0)

[error] Expected keyframe name "loadingGradientShift" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)